plenum/bls/bls_bft_replica_plenum.py

```diff
@@ -25,7 +25,7 @@ def __init__(self,
         self.state_root_serializer = state_roots_serializer
 
     def _can_process_ledger(self, ledger_id):
-        return ledger_id == DOMAIN_LEDGER_ID
+        return ledger_id != POOL_LEDGER_ID
```

Comment:
Please add a test for this

Reply:
New test: `plenum/test/bls/test_bls_bft_replica.py:test_process_ledger`
Updated test: `plenum/test/node_catchup/test_config_ledger.py:test_config_ledger_txns`. In this checked that config ledger stores state root in BLS sig store and state root hashes are the keys